generator/maxspeed_builder.cpp

```diff
@@ -0,0 +1 @@
+#pragma once
```

Comment:
?

Reply:
The only reason in dummy source file is to check self-consistency of the header. But in this case you need to replace
```c++
#pragma once
```
by
```
#include "generator/maxspeed_builder.hpp"
```

Reply:
The only reason in dummy source file is to check self-consistency of the header. But in this case you need to replace
```c++
#pragma once
```
by
```
#include "generator/maxspeed_builder.hpp"
```

Reply:
The only reason in dummy source file is to check self-consistency of the header. But in this case you need to replace
```c++
#pragma once
```
by
```
#include "generator/maxspeed_builder.hpp"
```

Reply:
The only reason in dummy source file is to check self-consistency of the header. But in this case you need to replace
```c++
#pragma once
```
by
```
#include "generator/maxspeed_builder.hpp"
```

Reply:
The only reason in dummy cpp file is to check ability of the header file to build. In this case you need to replace this pragma by
```c++
#include "maxspeed_builder.hpp"
```

Reply:
Thanks @mpimenov and @ygorshenin . Theses hpp and cpp files were committed by mistake. Now it's fixed.